integration/helpers/trustedclusters.go

```diff
@@ -141,3 +142,75 @@ func WaitForActiveTunnelConnections(t *testing.T, tunnel reversetunnel.Server, c
 		"Active tunnel connections did not reach %v in the expected time frame", expectedCount,
 	)
 }
+
+// TrustedClusterSetup is a grouping of configuration options describing the current trusted
+// clusters being tested used for passing info about the clusters to be tested to helper functions.
+type TrustedClusterSetup struct {
+	Ctx context.Context
```

Comment:
Can we avoid storing a context like this and instead pass it as an argument to long running functions?